HPCC-21721 Logical File popup inconsistent

miguelvazq opened the pull request:
Due to some refreshGrid code the DFUQuery was being called and therfore
the checkIfWarning method was being called. Added a flag to eliminate
calling this method more than once.

Signed-off by: Miguel Vazquez <miguel.vazquez@lexisnexisrisk.com>

<!-- Thank you for submitting a pull request to the HPCC project

 PLEASE READ the following before proceeding.

 This project only accepts pull requests related to open JIRA issues.
 If suggesting a new feature or change, please discuss it in a JIRA issue first.
 If fixing a bug, there should be an issue describing it with steps to reproduce.
 The title line of the pull request (and of each commit within it) should refer to the
 associated issue using the format:

 HPCC-nnnnn Short description of issue

 This will allow the Jira ticket to be automatically updated to refer to this pull request,
 and will ensure that the automatically-generated changelog is properly formatted.
 Where a pull request contains a single commit the pull request title will be set automatically,
 assuming that the commit has followed the proper guidelines.

 Please go over all the following points, and put an `x` in all the boxes that apply. You may find
 it easier to press the 'Create' button first then click on the checkboxes to edit the comment.
-->

## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [ ] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [ ] I have read the CONTRIBUTORS document.
- [ ] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [ ] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [ ] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [x] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Testing:
<!-- Please describe how this change has been tested.-->

<!-- Thank you for taking the time to submit this pull request and to answer all of the above-->

richardkchapman committed:
Community 7.0.22-rc1 Release Candidate 1

Signed-off-by: Richard Chapman <rchapman@hpccsystems.com>

richardkchapman committed:
Community 7.0.22-1 Gold

Signed-off-by: Richard Chapman <rchapman@hpccsystems.com>

miguelvazq commented:
@GordonSmith Please review.

HPCCSmoketest commented:
This PR (12337) is in the 2/2 position of the queue. Estimated start time is after ~1.0 hour(s)

HPCCSmoketest commented:
This PR (12337) is in the 1/1 position of the queue. Estimated start time is after ~0.5 hour(s)

HPCCSmoketest commented:
Process of PR-12337, label: LogicalFilePopup starts now.
The reason of this test is: Did not build yet.
Commit ID: 34200e40ed0410f39f2e2ac3795b39f471643fc8
Estimated completion time is ~0.5 hour(s)

HPCCSmoketest commented:
Automated Smoketest: :white_check_mark:
OS: centos 7.4.1708 (Linux 3.10.0-327.28.3.el7.x86_64)
Sha: 34200e40ed0410f39f2e2ac3795b39f471643fc8
Build: success
Build: success
Install HPCC Platform
HPCC Start: OK

Unit tests result:

| Test | total | passed | failed | errors | timeout | elaps | 
|---|---|---|---|---|---|---|
|unittest |95 |95 |0 |0 |0 |33 sec |
|wutoolTest(Dali) |19 |19 |0 |0 |0 |2 sec |
|wutoolTest(Cassandra) |19 |19 |0 |0 |0 |7 sec |



Regression test result:

| phase | total | pass | fail | elaps | 
|---|---|---|---|---|
|setup (hthor) |11 |11 |0 |27 sec  (00:00:27) |
|setup (thor) |11 |11 |0 |44 sec  (00:00:44) |
|setup (roxie) |11 |11 |0 |21 sec  (00:00:21) |
|test (hthor) |815 |815 |0 |248 sec  (00:04:08) |
|test (thor) |738 |738 |0 |591 sec  (00:09:51) |
|test (roxie) |889 |889 |0 |347 sec  (00:05:47) |


HPCC Stop: OK
Time stats:

| Prep time | Build time | Package time | Install time | Start time | Test time | Stop time | Summary | 
|---|---|---|---|---|---|---|---|
|   32 sec (00:00:32) |  188 sec (00:03:08) |    0 sec (00:00:00) |   64 sec (00:01:04) |   18 sec (00:00:18) | 1494 sec (00:24:54) |   18 sec (00:00:18) | 1814 sec (00:30:14) |

richardkchapman commented:
Please rebase onto 7.0.x

We can cherry-pick into appropriate release if/when it has passed review and been merged.

GordonSmith reviewed (approved):
Change looks fine

HPCCSmoketest commented:
Process of PR-12337, label: LogicalFilePopup starts now.
The reason of this test is: Base branch changed.
Commit ID: 34200e40ed0410f39f2e2ac3795b39f471643fc8
Estimated completion time is ~0.5 hour(s)

HPCCSmoketest commented:
Automated Smoketest: :x:
OS: centos 7.4.1708 (Linux 3.10.0-327.28.3.el7.x86_64)
Sha: 34200e40ed0410f39f2e2ac3795b39f471643fc8
Number of Error(s): 1
-- Configuring incomplete, errors occurred!
Number of Error(s): 1
-- Configuring incomplete, errors occurred!
CMakeFiles/CMakeOutput.log copied into ../CMakeOutput-2019-03-22_05-49-11.log

[Build error:](https://gist.github.com/eae21cfe331819cc0973c6e66a800014)
Build log [Build log](https://gist.github.com/HPCCSmoketest/eae21cfe331819cc0973c6e66a800014#file-result-19-03-22-05-48-38-log)

miguelvazq commented:
Fixing up this PR.

Commit:
HPCC-21721 Logical File popup inconsistent

Due to some refreshGrid code the DFUQuery was being called and therfore
the checkIfWarning method was being called. Added a flag to eliminate
calling this method more than once.

Signed-off by: Miguel Vazquez <miguel.vazquez@lexisnexisrisk.com>

Commit:
Delete version.cmake

miguelvazq commented:
Reopened here: https://github.com/hpcc-systems/HPCC-Platform/pull/12342
